README.md

```diff
@@ -0,0 +1,67 @@
+# Ruby Course
+
+If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+
+## Set-up Rubocop GitHub Action
+
+[Rubocop](https://www.rubocop.org/) is a Ruby static code analyzer (a.k.a. linter) and code formatter. It will enforce many of the guidelines outlined in the community [Ruby Style Guide](https://rubystyle.guide/).
+
+This GitHub Action is going to run [Rubocop](https://docs.rubocop.org/en/stable/) to help you find style issues.
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+    - **Remember** to use the file linked above
+    - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub Actions:
+
+![gh actions checks](../assets/images/gh-actions-rubocop-linters-checks.png)
+
+Click on the `Details` link to see the full output and the errors that need to be fixed:
```

Comment:
- [ ] please use [the Microverse template](https://github.com/microverseinc/readme-template) as an example of how your readme file should look like, the setup parts of the readme file should be related to how the user needs to set up your project instead of how to set up linters.

Reply:
Updated as requested. Thank you.